### ISSUE TITLE
Support PlantUML

### ISSUE THREAD

**Issue by PanderMusubi** [jbt/markdown-editor#34]
Please support PlantUML as https://stackedit.io/editor is doing.

**Comment by jbt** [comment]
Yeah good shout. UML diagrams and TeX (via [KaTeX](https://github.com/khan/katex)) are definitely things I want to add. Probably shouldn't be too hard.

**Comment by jbt** [comment]
Looks like stackedit uses https://bramp.github.io/js-sequence-diagrams/ - should be easy enough to integrate

**Comment by mlopezgva** [comment]
PlantUML is better (it covers 7 UML diagrams, and has many configurations options through `skinparams`). Some stackedit user made a js extension (and published a gist) to call plantuml online server and inject the image on the preview... Works like charm. M&darr; could benefit of that.
